applications/nrf5340_audio/doc/firmware_architecture.rst

```diff
@@ -7,14 +7,28 @@ nRF5340 Audio overview and firmware architecture
    :local:
    :depth: 2
 
-Each nRF5340 Audio application corresponds to one specific LE Audio role: unicast client (gateway), unicast server (headset), broadcast source (gateway), or broadcast sink (headset).
-The gateway receives the audio data from external sources (USB or line input/I2S) and forwards it to one or more headsets.
-The headset is a receiver device that plays back the audio it gets from the gateway, and will act as earbuds, headphones, a speaker, hearing aids, or similar.
+Each nRF5340 Audio application corresponds to one of two LE Audio modes and one of four specific LE Audio roles:
 
-Each nRF5340 Audio application is configured for one specific LE Audio mode: the *connected isochronous stream* (CIS, unicast) mode or in the *broadcast isochronous stream* (BIS) mode.
+* For the *connected isochronous stream* (CIS) mode:
+
+  * Unicast client
+  * Unicast server
+
+* For the *broadcast isochronous stream* (BIS) mode:
+
+  * Broadcast source
+  * Broadcast sink
+
+Unicast client (CIS) and broadcast source (BIS) receive the audio data from external sources (USB or line input/I2S) and forward it to one or more unicast servers or broadcast sinks, thus becoming source devices for them.
+Unicast servers (CIS) and broadcast sinks (BIS) are receiver devices that play back the audio they get from source devices, and will act as earbuds, headphones, speakers, hearing aids, or similar.
```

Comment:
This is where we may rephrase a bit. A broadcast sink is always a receiver, but a unicast server can be a sink and a source.

Reply:
I think this is clarified in line 24?